pkg/util/sloconfig/config.go

```diff
@@ -18,14 +18,22 @@ package sloconfig
 
 import "flag"
 
+const (
+	Zh = "zh"
+	En = "en"
+)
+
 var (
 	// ConfigNameSpace is the namespace of the slo-controller configmap.
 	ConfigNameSpace = "koordinator-system"
 	// SLOCtrlConfigMap is the name of the slo-controller configmap.
 	SLOCtrlConfigMap = "slo-controller-config"
+	// DefaultTranslator = "en"
+	DefaultTranslator = "en"
 )
 
 func InitFlags(fs *flag.FlagSet) {
 	fs.StringVar(&SLOCtrlConfigMap, "slo-config-name", SLOCtrlConfigMap, "determines the name the slo-controller configmap uses.")
 	fs.StringVar(&ConfigNameSpace, "config-namespace", ConfigNameSpace, "determines the namespace of configmap uses.")
+	fs.StringVar(&DefaultTranslator, "default_translator", DefaultTranslator, "determines the sloConfig validator translator.")
```

Comment:
how about (`..., "slo-config-translator", DefaultTranslator, "determines the sloConfig validator translator. e.g. 'en', 'zh'")`?